packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -771,7 +782,10 @@ class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProvider
                 ?? expansionTileTheme.expandedAlignment
                 ?? Alignment.center,
               heightFactor: _heightFactor.value,
-              child: child,
+              child: ColoredBox(
```

Comment:
I think we should add the `childrenBackgroundColor` in the `Material` widget you added below, instead of here:) This ColoredBox here looks redundant:)

Reply:
Your proposal does not address the problem because what we actually need is to add a color to the padding area. Therefore, I suggest renaming the variable from `childrenBackgroundColor` to `childrenPaddingColor`, i think this the right way.
I used the `ColoredBox` widget to apply a color to the entire children area, and then wrapped the children with a `Material` widget to isolate them from this color